app/models/vm_scan.rb

```diff
@@ -600,6 +556,32 @@ def abort_retry(*args)
 
   private
 
+  def create_snapshot(vm)
```

Comment:
@roliveri Are you referring to returning true on line [578](https://github.com/ManageIQ/manageiq/pull/9419/files#diff-dcc9d6b083539d15b731c9262766322bR578)?